<launch>
    <arg name="points_src" default="/mmWaveDataHdl/RScan" /> <!-- PointCloud source topic-->
    <arg name="image_src" default="/rasp_cam/image_raw/compressed" /> <!-- Raw Image source topic to be rectified-->
    <arg name="camera_info_src" default="/rasp_cam/camera_info" /> <!-- CameraInfo source topic-->

    <node name="cam_radar_fusion_01" pkg="cam_radar_fusion" type="cam_radar_fusion" output="screen">
        <param name="points_src" value="$(arg points_src)" />
        <param name="image_src" value="$(arg image_src)" />
        <param name="camera_info_src" value="$(arg camera_info_src)" />
    </node>

</launch>